ocs_ci/ocs/hsbench.py

```diff
@@ -134,35 +152,202 @@ def run_benchmark(
+        ) as out:
+            writer = csv.writer(out)
+            for row in csv.reader(file):
+                if row[1] == "TOTAL":
+                    writer.writerow(row)
+
+        with open(f"{self.warp_dir}/summary.csv", "r") as read_obj:
+            reader = csv.reader(read_obj)
+            for row in reader:
+                eval_data[row[3]] = row[4]
+
+        if put is True:
+            put_val = eval_data["PUT"]
+            log.info(f"Number of PUT objects is {put_val}")
+            if put_val != num_objs:
+                assert put_val, (
+                    "Number of PUT objects don't match with number objects."
+                    f"Expecting {num_objs} but getting {put_val}"
+                )
+        if get is True:
+            get_val = eval_data["GET"]
+            log.info(f"Number of GET objects is {get_val}")
+            if get_val != num_objs:
+                assert get_val, (
+                    "Number of GET objects don't match with number objects."
+                    f"Expecting {num_objs} but getting {get_val}"
+                )
+        if list_obj is True:
+            list_val = eval_data["LIST"]
+            log.info(f"Number of LIST objects is {list_val}")
+            if list_val != 0:
+                assert list_val, (
+                    "LIST objects doesn't show correctly."
+                    f"Expecting LIST objects is not zero but getting {list_val}"
+                )
+
+    def delete_objects_in_bucket(self, bucket_name=None):
+        """
+        Delete objects in a bucket
+
+        Agr:
+            bucket_name (str): Name of bucket
+
+        """
+        log.info("Deleting objects in bucket...")
+        self.toolbox.exec_sh_cmd_on_pod(
+            f"radosgw-admin object rm --object=000000000000 --bucket={bucket_name}"
+        )
+        num_object = self.toolbox.exec_sh_cmd_on_pod(
+            f"radosgw-admin bucket stats --bucket={bucket_name} | grep num_objects"
+        )
+        log.info(f"Number objects in {bucket_name} is {num_object}")
+        if num_object == (self.num_obj - 1):
+            log.info(f"Number objects in {bucket_name} is {num_object}")
+        assert num_object, (
+            f"Number object in bucket {bucket_name} is {num_object}"
+            f"Expecting {self.num_obj - 1} in {bucket_name}"
+        )
+
+    def delete_bucket(self, bucket_name=None):
+        """
+        Delete bucket
+
+        Agr:
+            bucket_name (str): Name of bucket
+
+        """
+        log.info("Deleting bucket with objects...")
+        self.toolbox.exec_sh_cmd_on_pod(
+            f"radosgw-admin bucket rm --bucket={bucket_name} --purge-objects"
+        )
+        num_bucket = self.toolbox.exec_sh_cmd_on_pod(
+            f"radosgw-admin bucket list | grep {self.bucket_prefix} | wc -l"
+        )
+        log.info(f"Number buckets are {num_bucket}")
+        log.info(f"Expecting buckets are {self.num_bucket - 1}")
```

Comment:
It would be better if we just validated the deleted bucket is listed, instead of verifying bucket count is reduced after delete. wdyt?

Reply:
It is doable.  I can look into this.

Reply:
This can be addressed in the next PR.